### ISSUE TITLE
XMPP chat error, plus won't let me login (unexpected error http 400)

### ISSUE THREAD

**Issue by GitHubGuy89** [tomlandia/fuckr#64]
Using latest version as of 8/12/2016. At first I was able to send chat messages, then it stopped letting me.

Also keep receiving a popup error stating:
"XMPP chat error. If you're using public wifi, XMPP protocol is probably blocked."

I tried to reinstall and when I went to login to my account, it now gives me a new popup error:
"Unexpected error (HTTP 400)"

When I close that, the desktop app has a window saying:
"Requests to the server have been blocked by an extension."

Update: It seems the accounts I created within the new version are getting suspended by grindr, which is the reason for the Unexpected Error. There is nothing "spamy/suspend-able" about the accounts other then I created them using fuckr and changed my location. Possibly they are onto account creation using fuckr? I am able to log into my old account fine. May try to create an account on the phone app, and log in using fuckr to see if it gets suspended also.

This is Windows x64 desktop version.

**Comment by shyrat** [comment]
Your account is banned if while logging in (POSTing a request to /2.0/sessions), the server returns HTTP/400 with a response {"code":1001}.  Not all HTTP/400 are bans -- for example, code 1101 means your password is wrong.

(Most people aren't expected to know what that means, but Thomas could use this to make Fuckr give a specific error message for banned accounts.)

**Comment by GitHubGuy89** [comment]
Yup definitely a ban. Created new issue for that. Will keep this up for the XMPP chat error problem that happens.

**Comment by shyrat** [comment]
For what it's worth, I just tried creating accounts with:
- the old 2.8.x official Grindr app
- my own unofficial Grindr app
both of which use the same v2 account creation APIs that Fuckr uses.  In both cases, my account did not seem to be banned.

One possibility is that a spammer used the same IP address that you used, so your network is just blocked.

If you need to create a new account and you don't have an Andriod/iOS phone handy, you can
1. Open this in a web browser: https://neo-account.grindr.com/signup?clientVersion=9 
2. Fill out all the fields and click submit.
3. If it doesn't show any error and instead just appears to sit there, then it worked.  (Normally the app intercepts the success case, so the website doesn't bother to show anything if it works.)
4. Open your third party Grindr client and use its flow to log into an existing account.

**Comment by GitHubGuy89** [comment]
Created account on my PC via that link, but created the actual profile thru the mobile app. Then logged into Fuckr desktop app, and has been working without getting banned. Don't plan on updating the profile thru the desktop app though, just in case.

Have noticed that I can not upload photos thru the desktop app anymore though to send in messages. Has the + sign to add an image, but once I select the image it just goes back to the message screen without anything happening or uploading.

**Comment by tomlandia** [comment]
Replaced by #66.
